configuration/azure/appconfig/appconfig.go

```diff
@@ -159,33 +159,80 @@ func parseMetadata(meta configuration.Metadata) (metadata, error) {
 
 func (r *ConfigurationStore) Get(ctx context.Context, req *configuration.GetRequest) (*configuration.GetResponse, error) {
 	keys := req.Keys
-	items := make([]*configuration.Item, 0, len(keys))
-	for _, key := range keys {
-		resp, err := r.client.GetSetting(
-			ctx,
-			key,
-			&azappconfig.GetSettingOptions{
-				Label: to.Ptr("label"),
-			})
-		if err != nil {
-			return nil, err
-		}
+	var items []*configuration.Item
 
-		item := &configuration.Item{
-			Metadata: map[string]string{},
+	if len(keys) == 0 {
+		var err error
+		if items, err = r.getAll(ctx); err != nil {
+			return &configuration.GetResponse{}, err
 		}
-		item.Key = key
-		item.Value = *resp.Value
-		item.Metadata["label"] = *resp.Label
-
-		items = append(items, item)
+	} else {
+		items = make([]*configuration.Item, 0, len(keys))
+		i := 0
+		for _, key := range keys {
+			resp, err := r.client.GetSetting(
+				ctx,
+				key,
+				&azappconfig.GetSettingOptions{
+					Label: to.Ptr("label"),
```

Comment:
By setting this value, we are only querying keys that have the label as `label`. For all the others only an error is returned. This either should be a filed read from the request metadata or not used at all.

Reply:
got the response like following when the Label `label` is not present for the key
````fail to get [test-dapr] from Configuration store test-azappconfig: GET https://{instance}.azconfig.io/kv/test-key\n--------------------------------------------------------------------------------\nRESPONSE 404: 404 Not Found\nERROR CODE UNAVAILABLE\n--------------------------------------------------------------------------------\nResponse contained no body\n--------------------------------------------------------------------------------\n"```

Reply:
At the same time when a request is made for a key `test-lab` which has the Label `label` the output is the following:
```
[
    {
        "key": "test-lab",
        "value": "test-lab",
        "metadata": {
            "label": "label"
        }
    }
]
```

Reply:
Should be `Label: to.Ptr("*")`